tartufo/types.py

```diff
@@ -39,6 +39,13 @@ class Chunk:
     metadata: Dict[str, Any] = field(default_factory=dict)
 
 
+@dataclass
+class Rule:
```

Comment:
I'm wondering if it makes sense to use a more explicit name here. In the scanners, we call these `regex_rules`, which is more of a legacy hold-over than anything else. I'm just concerned that `Rule` is very generic and could be easily misconstrued. What about something like... `RegexRule` (keeping with the previous naming), `MatchRule`... those are the only ones coming to mind right now.